UnityMcpBridge/UnityMcpServer~/src/tools/manage_script.py

```
@@ -558,8 +568,10 @@ def manage_script(
 
     @mcp.tool(description=(
         "Get manage_script capabilities (supported ops, limits, and guards).\n\n"
+        "Args:\n- random_string: required parameter (any string value)\n\n"
         "Returns:\n- ops: list of supported structured ops\n- text_ops: list of supported text ops\n- max_edit_payload_bytes: server edit payload cap\n- guards: header/using guard enabled flag\n"
     ))
+    @telemetry_tool("manage_script_capabilities")
     def manage_script_capabilities(ctx: Context) -> Dict[str, Any]:
```

Ruff (0.12.2)

Unused function argument: `ctx`

(ARG001)